Client/Frontend/Browser/TabTrayController.swift

```diff
@@ -223,9 +225,17 @@ struct PrivateModeStrings {
     static let toggleAccessibilityValueOff = NSLocalizedString("Off", tableName: "PrivateBrowsing", comment: "Toggled OFF accessibility value")
 }
 
+protocol TabTrayDelegate {
+    func tabTrayDidDismiss(tabTray: TabTrayController)
+    func addBookmark(tab: Browser)
+    func addToReadingList(tab: Browser) -> ReadingListClientRecord?
+    func present(viewController viewController: UIViewController)
```

Comment:
Can we prefix these with `tabTrayX` (`tabTrayDidAddBookmark`, `tabTrayDidAddToReadingList`, `tabTrayDidPresentViewController`, or something similar)? It's helpful when implementing these protocols via autocomplete and makes it easier to see where they came from.